### PR TITLE
Move info from README

### PR DESCRIPTION
**Pull request by RomanTsukanov**
Merge only after Vue docs are available on js.devexpress.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,495 +5,21 @@
 
 ### If you are looking for the v18.2 branch, please follow this link: [https://github.com/DevExpress/devextreme-vue/tree/18.2](https://github.com/DevExpress/devextreme-vue/tree/18.2).
 
-This project allows you to use [DevExtreme](http://js.devexpress.com) [Vue](https://vuejs.org) Components.
+This project allows you to use [DevExtreme widgets](http://js.devexpress.com/Demos/WidgetsGallery/) in [Vue](https://vuejs.org) applications.
```
Comment (nightskylark) [inline README.md]: We have decided to use the `component` word in such context.

File: README.md [hunk 2]
```diff
@@ -5,495 +5,21 @@
 
 ### If you are looking for the v18.2 branch, please follow this link: [https://github.com/DevExpress/devextreme-vue/tree/18.2](https://github.com/DevExpress/devextreme-vue/tree/18.2).
```
Comment (nightskylark) [inline README.md]: We don not need this note in case the Readme does not contain any documentation.

File: README.md [hunk 3]
```diff
@@ -5,495 +5,21 @@
...
 
-This project allows you to use [DevExtreme](http://js.devexpress.com) [Vue](https://vuejs.org) Components.
+This project allows you to use [DevExtreme widgets](http://js.devexpress.com/Demos/WidgetsGallery/) in [Vue](https://vuejs.org) applications.
 
+* [Documentation](https://js.devexpress.com/Documentation/Guide/Vue_Components/DevExtreme_Vue_Components/)
+* [Technical Demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/Vue/Light/)
+* [API Reference](http://js.devexpress.com/Documentation/ApiReference/)
+* [Live example](https://codesandbox.io/s/github/lukyanovas/devextreme-vue-example)
```
Comment (nightskylark) [inline README.md]: How about to add the link to the App Template here?

File: README.md [hunk 4]
```diff
@@ -5,495 +5,21 @@
 
 ### If you are looking for the v18.2 branch, please follow this link: [https://github.com/DevExpress/devextreme-vue/tree/18.2](https://github.com/DevExpress/devextreme-vue/tree/18.2).
```
Comment (nightskylark) [inline README.md]: Let's add a beautiful picture here. Sergey Z. have agreed that it's a good idea.

Comment (RomanTsukanov) [reply README.md]: Let's do this in another PR.

**Pull request merged**